src/nvs.rs

```diff
@@ -736,6 +805,80 @@ impl RawHandle for EspNvs<NvsEncrypted> {
         self.1
     }
 }
+impl RawHandle for EspNvs<NvsDefault> {
+    type Handle = nvs_handle_t;
+
+    fn handle(&self) -> Self::Handle {
+        self.1
+    }
+}
+
+#[cfg(esp_idf_version_at_least_5_2_0)]
+pub struct EspNvsKeys<'a> {
+    // The EspNvs must not be dropped while the iterator is still in use,
+    // this reference ensures that.
+    _nvs: PhantomData<&'a ()>,
+    raw_iter: nvs_iterator_t,
+    is_exhausted: bool,
+    key_name_buffer: [u8; 16],
```

Comment:
Why 16 and not 11?

Reply:
They key buffer in nvs_info is 16 byte large

Reply:
See https://docs.esp-rs.org/esp-idf-sys/esp_idf_sys/struct.nvs_entry_info_t.html